### ISSUE TITLE
Pending Balance - show how much in another currency

### ISSUE THREAD

**Issue by JokerGermany** [zone117x/node-cryptonote-pool#186]
Feature Request:
It would  be cool if i could choose how much the pending balance is worth in another currency like euro.

**Comment by mikel-fernandez** [comment]
Good idea, I implemented it, see it in commit 134d6eaf6402dd02a58a8bc9b65694b63849773e
